### PR TITLE
BETA Stale Feature Docs

### PR DESCRIPTION
**Pull request by katemacfarlane**
stale feature docs for beta release

### CODE REVIEW

File: docs/platform/feature-flags/stale-feature-notifications.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+Features can belong to one of the following staleness reaons:
+
+### Unmodified - Short-Lived 
+
+A Feature is classified as `Unmodified - Short-Lived` if it has not been updated for more than 1 month.
+
+**Applicable to**: Release, Experiment Feature types.
+
+### Unmodified - Long-Lived
```
Comment (kaushalkapasi) [inline docs/platform/feature-flags/stale-feature-notifications.md:24]: should we merge both `Unmodified` into a single section and have sub-headers by Feature Type instead?

Comment (cdruxerman) [reply docs/platform/feature-flags/stale-feature-notifications.md:24]: also, the timelines stated here are the same, just in different units.

short-lived is stated as unmodified for 1 month and long-lived for 30 days.

File: docs/platform/feature-flags/stale-feature-notifications.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+### Released
+
+A **Released** Feature is one that has been serving the same variation to all users in a production environment for 14 or more days. DevCycle will check that the distribution has reached 100% and all rollouts are complete. 
+
+**Applicable to**: Release, Experiment Feature types.
+
+### Unused 
+
+An **Unused** Feature is one where there are no evaluations or defaults for any variables in the feature for 2 weeks. Targeting status is irrelevant.
```
Comment (cdruxerman) [inline docs/platform/feature-flags/stale-feature-notifications.md:38]: "for any Variables associated with the Feature"

File: docs/platform/feature-flags/stale-feature-notifications.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+![image of list page](/nov-2024-stale-feat-list-page.png)
+
+## Snoozing & Disabling Stale Feature Notifications 
+
+If a Feature is marked as stale, you will see a notification at the top of the Feature page.
+
+Click the `Details` button on the notification. This will take you to the Status section of the Feature page, where you will find the specific reason for the Feature's staleness.
+
+To snooze a notification for a Feature, cclick on the `Snooze` button and select a timeframe in which you'd like to pause staleness checks for this Feature. After the time has elapsed, the Feature will be checked for staleness again.
```
Comment (cdruxerman) [inline docs/platform/feature-flags/stale-feature-notifications.md:74]: to snooze.... click on the

There's an extra c on click

**Review by cdruxerman — COMMENTED** [review]
I left a couple comments. In general I think it's ok for beta, but I think it is missing a section on what we think a team's process should look like when staleness notifications are a part of their workflow

**Comment by katemacfarlane** [comment]
> I left a couple comments. In general I think it's ok for beta, but I think it is missing a section on what we think a team's process should look like when staleness notifications are a part of their workflow

@cdruxerman  - yeah agreed, i still need to update the feature lifecycle pages and reference them back to this as well!

**Review by kaushalkapasi — APPROVED** [review]

**Pull request merged**